src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -163,14 +185,69 @@ public static void TcpClientCredentialType_Certificate_EchoString()
 
             serviceProxy = factory.CreateChannel();
 
+            // *** EXECUTE *** \\
             string result = serviceProxy.Echo(testString);
+
+            // *** VALIDATE *** \\
             Assert.Equal(testString, result);
 
+            // *** CLEANUP *** \\
             ((ICommunicationObject)serviceProxy).Close();
             factory.Close();
         }
         finally
         {
+            // *** ENSURE CLEANUP *** \\
+            ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
+        }
+    }
+
+    [Fact]
+    [ActiveIssue(485)]
+    [OuterLoop]
+    public static void TcpClientCredentialType_Certificate_CustomValidator_EchoString()
+    {
+        string clientCertThumb = null;
+        EndpointAddress endpointAddress = null;
+        string testString = "Hello";
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
+
+        try
+        {
+            // *** SETUP *** \\
+            NetTcpBinding binding = new NetTcpBinding(SecurityMode.Transport);
+            binding.Security.Transport.ClientCredentialType = TcpClientCredentialType.Certificate;
+
+            endpointAddress = new EndpointAddress(new Uri(Endpoints.Tcp_ClientCredentialType_Certificate_CustomValidation_Address),
+                new DnsEndpointIdentity(Endpoints.Tcp_VerifyDNS_HostName));
+            clientCertThumb = BridgeClientCertificateManager.LocalCertThumbprint; // ClientCert as given by the Bridge
+
+            factory = new ChannelFactory<IWcfService>(binding, endpointAddress);
+            factory.Credentials.ServiceCertificate.Authentication.CertificateValidationMode = X509CertificateValidationMode.None;
```

Comment:
Authentication.CertificateValidationMode = Custom, not None to hook up your own validator. That's why the test fails.